src/lib/holocene/modal.svelte

```diff
@@ -67,12 +95,13 @@
     />
     <div
       bind:this={modalElement}
-      class="body"
+      class="body {className}"
       class:large
       tabindex="-1"
       role="alertdialog"
       aria-labelledby="modal-title"
       aria-describedby="modal-description"
```

Comment:
Looks like we could also add an [aria-modal="true](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal)" here.